### DISCUSSION TITLE
Add a logo in the first page

### DISCUSSION THREAD

**Discussion by ler93** [jspsych/jsPsych#2083] [Q&A] [answered]
Hi! I'm trying to add a logo of the institution that funds the study to the first page of my experiment. Since the first page is a "fullscreen" type, none of the available parameters in this kind of plugin allows me to add the image. I checked the jspsych page but couldn't find the solution. I hope someone can help me. Thanks in advance!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @ler93,

You can insert an image using the `message` [parameter](https://www.jspsych.org/plugins/jspsych-fullscreen/#parameters) in the full screen plugin:

```js
var trial = {
  type: 'fullscreen',
  message: `<p><img src="funders-logo.png"></img></p>
    <p>The experiment will enter full screen mode when you click the button below.</p>`
}
```

**Reply by ler93** [reply]
Thank you, @jodeleeuw! I had not realized that this could be a way to do it. It worked perfectly.
